content/ADR-204-comms-architecture.md

```diff
@@ -39,6 +40,7 @@ The Archipelago protocol scales efficiently due to its dynamic island calculatio
 Maintain the current Archipelago implementation while establishing a new scene room connection specific to the active scene. The Archipelago channel should be reserved solely for receiving avatar positions, profile update notifications, and nearby chat messages with the objective to be able to interact with users beyond the limits of a scene. All communications, including voice chat, scene objects state, and streams, will be shared within the scene channel. 
 This new model grants scene owners authority over the communication channel, allowing them to share content streams, share object states or handling speakers, providing a consistent experience for all users visiting the scene.
 
+At any given moment, a user can maintain up to two connections: one with the current scene and another with the island. When the scene changes, the existing connection to the previous scene should be closed, and a new connection to the current scene must be established. 
```

Comment:
Is there a specific reason to limit to only one scene connection? Wouldn't it be up to the explorer to determine how many scene-rooms can handle?

Reply:
Ultimately, it is the client’s responsibility to manage the number of open connections. However, keep in mind that maintaining multiple open connections is resource-intensive, and the data transmitted over the scene channel can require substantial bandwidth to share scene state and other streams. It is advisable to keep a connection open only with the scene that the user is currently interacting with.